packages/mui-material/src/styles/experimental_extendTheme.test.js

```diff
@@ -59,6 +59,9 @@ describe('experimental_extendTheme', () => {
     expect(theme.colorSchemes.dark.palette.background.defaultChannel).to.equal('18 18 18');
     expect(theme.colorSchemes.light.palette.background.defaultChannel).to.equal('255 255 255');
 
+    expect(theme.colorSchemes.dark.palette.background.paperChannel).to.equal('18 18 18');
+    expect(theme.colorSchemes.light.palette.background.paperChannel).to.equal('255 255 255');
```

Comment:
Out of curiosity, why does this new `paperChannel` token have the same colors as `defaultChannel`? What's the difference?

Reply:
It's because the default values of `paper` and `default` are the same.